python/src/aiconfig/editor/client/src/components/aiconfigReducer.ts

```diff
@@ -78,6 +80,13 @@ export type SetNameAction = {
   name: string;
 };
 
+export type StreamAIConfigChunkAction = {
+  type: "STREAM_AICONFIG_CHUNK";
+  id: string;
+  cancellationToken?: string;
+  isRunning?: boolean;
```

Comment:
With this new action, we should be able to infer isRunning in the reducer now instead of passing it through the action, right?

Reply:
Yes correct, this diff is all just copy pasting to keep it easier to review and see the steps I did!